rfc/001-composable-pipelines/README.md

```diff
@@ -0,0 +1,360 @@
+# Composable Pipelines
+
+- **Status**: In-Review
+- **Created**: Aug 19, 2022
+- **ETA**: Sep 21, 2022
+- **Authors**:
+  - [Matthias Vallentin](https://github.com/mavam)
+- **Contributors**:
+  - [Anthony Verez](https://github.com/netantho)
+  - [Dominik Lohmann](https://github.com/dominiklohmann)
+  - [Rémi Dettai](https://github.com/rdettai)
+  - [Thomas Peiselt](https://github.com/dispanser)
+- **Discussion**: [PR #2511](https://github.com/tenzir/vast/pull/2511)
+
+## Overview
+
+This proposal ideates a new UX around VAST's pipelines. The fundamental idea is
+that *everything is a pipeline* and build a composable UX around this concept.
+
+## Problem Statement
+
+Pipelines currently are only deployable at [predefined
+points](https://vast.io/docs/use-vast/transform). For example, they can be
+triggered for specific events on import/export, or invoked by compaction.
```

Comment:
I think we can make the motivation more explicit by defining the concept of a pipeline in this piece.

```suggestion
With transforms, VAST provides a powerful mechanism to modify and reshape
data at various points in their lifetime, be it during ingestion, before sending it
to clients, or as part of the compaction mechanism. What is not possible however
is the application of data transforms in an ad-hoc manner, for example as a post-
processing step of a data export or for an ad-hoc ingest of a fixed data set.

In this RFC we propose the **pipeline** query format. It is a dynamic way
to access the transforms feature that used to be restricted to the configuration
file and thereby reserved to the database administrator until now.
```

Reply:
Transforms are no longer a thing since https://vast.io/blog/vast-v2.2#transforms-are-now-pipelines. It would be confusing to jump back to this terminology

Can you take a look at 119a3659d and resolve this thread if you like the way I integrated it?